### ISSUE TITLE
Not able to see predictions in Image project section.

### ISSUE THREAD

**Issue by globulartorpedo** [googlecreativelab/teachablemachine-community#121]
I am unable to see the predictions as it is outside the card element , Please fix.
![Screenshot from 2020-07-18 09-11-13](https://user-images.githubusercontent.com/45914394/87844095-58b4f700-c8d7-11ea-8034-87fac25f9b22.png)

**Comment by leonardo-beabr** [comment]
Have you tried to decrease the screen size?

**Comment by globulartorpedo** [comment]
Yes , I have tried it and I think it is an issue with the front end.

**Comment by EmkaMk** [comment]
Yes, I thought it was my screen but its an issue on their end.

**Comment by EmkaMk** [comment]
Hello, is someone working on this? It is a big issue if you have more than 3 classes, you are unable to see the predictions. How am I supposed to train the model if I cant see what is the output? The card element is not scrollable.

![image](https://user-images.githubusercontent.com/9258377/96699327-6e323500-138e-11eb-97dd-3d83f4d81134.png)
